governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,66 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3]
+    abstain: [0]
+  victory_conditions:
+    - { 
+        type: 'and', 
+        conditions: [
+          { type : approval },
+          { type: majority, percent: 50 },
+        ]
+      }
+    - { type : default, value : $StatusQuoOrRejectOptionNumber }
+  result_display: approval-breakdown
+options:
+   0: Abstain
+   1: $Option1
+   2: $Option2
+   3: $Option3
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Poll:$GenericChange - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should select all the options that you would support in an executive vote.**
```

Comment:
'No Change' needs to always be an option for these. Makes this a little misleading. We might have the same problem elsewhere though.

Reply:
(Unless we're disregarding the majority condition)